src/main/scala/mockws/FakeAhcResponse.scala

```diff
@@ -47,13 +47,17 @@ class FakeAhcResponse(result: Result, body: Array[Byte]) extends Response {
   override def isRedirected: Boolean = Set(301, 302, 303, 307, 308).contains(getStatusCode)
 
   override def getCookies: util.List[Cookie] = {
+    val shouldCookieBeWrappedInQuotes = false
     result.newCookies.map(playCookie => new Cookie(
       playCookie.name,
       playCookie.value,
-      false, // wrap = should the value be wrapped in quotes
+      shouldCookieBeWrappedInQuotes,
```

Comment:
sorry, I missed that we were calling java here... No named parameters possible... 😭